.env.example

```diff
@@ -1,16 +1,13 @@
 APP_ENV=development # or qa or production
 
-#IIIF 
-IIIF_URL=https://iiif.nypl.org
-
 # Repo api
 API_URL=https://api.repo.nypl.org # must be https
 AUTH_TOKEN= # Get credentials from a parameter store or a teammate!
 API_DEBUG=true # Optional, controls verbosity of api related log output
 API_CACHE=false  # Optional, controls whether or not api responses are cached
 
-# Old DC url
-DC_URL='' # OR https://qa-digitalcollections.nypl.org OR https://digitalcollections.nypl.org
+# Base DC url
+# DC_URL=''
```

Comment:
This is removed in the `ENVIRONMENTVARS.md` file, so it should be removed here too. I don't see any references to it anymore.